package.json

```diff
@@ -174,7 +174,7 @@
     "chalk": "^4.1.1",
     "ci-info": "^3.3.2",
     "classnames": "~2.2.6",
-    "crypto-js": "~3.3.0",
+    "crypto-js": "^4.1.1",
```

Comment:
Think I'm happy with changing this to caret to pick up the minor releases. Good change.